web/ui/react-app/src/themes/light.scss

```diff
@@ -0,0 +1,15 @@
+@import '~@forevolve/bootstrap-dark/dist/css/toggle-bootstrap.min.css';
+
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+
+$config-yaml-color: #333;
+$config-yaml-bg: $jumbotron-bg;
```

Comment:
Rather than repurposing `$jumbotron-bg` (whose name is confusing in this context, and whose color is a bit darker than what we used here previously), I think let's use a custom color?